docs/tutorials/sinks/1_basic_sink.md

```diff
@@ -280,11 +281,8 @@ Add the following after emitting `BytesSent`:
 
 ```diff
 +     let event_byte_size = event.estimated_json_encoded_size_of();
-+     emit!(EventsSent {
-+         count: 1,
-+         byte_size: event_byte_size,
-+         output: None,
-+     });
++     let events_sent = register!(EventsSent::from(Output(None)));
++     events_sent.emit(CountByteSize(1, event_byte_size));
```

Comment:
Similar to the comment above, it would be helpful to show the diff for the entire `run_inner` function and to place `let events_sent = register!(EventsSent::from(Output(None)));` outside of the for loop.